### DISCUSSION TITLE
Metamask support in web3Modal for react native

### DISCUSSION THREAD

**Discussion by ArslanHaq** [WalletConnect/walletconnect-monorepo#2463] [Developer Support] [answered]
I had integrated the web3modal in my React native app but in the modal Metamask wallet option is not available, any type of help is appreciated

**Comment by finessevanes** [comment, marked as answer]
Hey - MetaMask will be supporting v2 soon. The newest [React Native Package](https://docs.walletconnect.com/2.0/reactnative/web3modal/dapp-usage) only supports v2 wallets.
